src/mbgl/map/transform.cpp

```diff
@@ -313,18 +324,24 @@ void Transform::_easeTo(const CameraOptions& options, double new_scale, double n
     }
 }
 
+/**
+* Flying animation to a specified location/zoom/bearing with automatic curve.
+*/
 void Transform::flyTo(const CameraOptions &options) {
+
     CameraOptions flyOptions(options);
     LatLng latLng = options.center ? *options.center : getLatLng();
     LatLng startLatLng = getLatLng();
     double zoom = flyOptions.zoom ? *flyOptions.zoom : getZoom();
     double angle = flyOptions.angle ? *flyOptions.angle : getAngle();
     double pitch = flyOptions.pitch ? *flyOptions.pitch : getPitch();
+
     if (std::isnan(latLng.latitude) || std::isnan(latLng.longitude) || std::isnan(zoom)) {
         return;
     }
     
     double new_scale = std::pow(2.0, zoom);
+    mbgl::Log::Info(mbgl::Event::JNI, "flyTo - new_scale: %f", new_scale);
```

Comment:
Just saw your note in the page refresh after testing and merging.  I'll remove that next.